### PR TITLE
fixed the create-new tab issue after applying filter - 1743

### PR DESCRIPTION
**Pull request by Arti-Khillare1**

### CODE REVIEW

File: projects/lib-shared-modules/src/lib/constants/urlConstants.ts [hunk 1]
```diff
@@ -16,7 +16,11 @@ export const reviewStatus = {
 
 };
 export const ROUTE_PATHS = {
+    CREATE_NEW : {
```
Comment (ramkumar-ns-dev) [inline projects/lib-shared-modules/src/lib/constants/urlConstants.ts]: why new object here ?

File: projects/lib-shared-modules/src/lib/constants/urlConstants.ts [hunk 2]
```diff
@@ -16,7 +16,11 @@ export const reviewStatus = {
...
+      CREATE_NEW: 'create-new'
+    },
     SIDENAV: {
+      BROWSE_EXISTING: 'browse-existing',
```
Comment (ramkumar-ns-dev) [inline projects/lib-shared-modules/src/lib/constants/urlConstants.ts:20]: why is this here ? are you working on browse-existing ?

File: projects/self-creation-portal/src/app/services/common-service/common.service.ts [hunk 3]
```diff
@@ -56,23 +56,33 @@ export class CommonService {
...
-      this.router.navigate([], {
+    (!istargetpaths && Object.keys(queryParams).length > 0)) {
+      this.router.navigate([], {  
         relativeTo: this.route,
         queryParams: {}
+      }).then(() => {
+      // After clearing query params, navigate to "create-new" if applicable
+      if (isCreateNew) {
+        setTimeout(() => {
+          this.router.navigate([createNewPath], { relativeTo: this.route });
```
Comment (ramkumar-ns-dev) [inline projects/self-creation-portal/src/app/services/common-service/common.service.ts]: why setTimeout for route change ? please explain the use case of setTimrout here

**Review by ramkumar-pacewisdom — CHANGES_REQUESTED** [review]

**Review by ramkumar-pacewisdom — APPROVED** [review]
